RandLAPACK/drivers/rl_cqrrpt.hh

```diff
@@ -285,7 +285,7 @@ int CQRRPT<T, RNG>::call(
     for(i = 0; i < k; ++i) {
         curr_entry = std::abs(R_sp[i * k + i]);
         if(curr_entry > running_max) running_max = curr_entry;
-        if(curr_entry < running_min) running_max = running_min;
+        if(curr_entry < running_min) running_min = running_max;
```

Comment:
This is confusing. Why are you setting the running min to the running max? It should be ``running_min = curr_entry`` if this path is hit.

Better yet, just do
```
running_max = std::max(running_max, curr_entry);
running_min = std::min(running_min, curr_entry);
```

Reply:
yeah yeah